.github/workflows/vast.yaml

```diff
@@ -126,7 +149,7 @@ jobs:
       CXX: ${{ matrix.build.cxx }}
       CCACHE_ABSSTDERR: true
       CCACHE_COMPRESS: true
-      CCACHE_COMPRESSLEVEL: 9
+      CCACHE_COMPRESSLEVEL: 0
```

Comment:
This seems like an unrelated change; I'm fine with it, just making sure it's intended

Reply:
It is intended. Just forgot to mention it in the description, but a recent release of ccache made level 0 much smarter with regards to the size limit, so it's better to use this now. I just remembered that I wanted to make this change when I worked on this PR.